modules/synacormediaBidAdapter.js

```diff
@@ -79,41 +100,43 @@ export const spec = {
     }
   },
   interpretResponse: function(serverResponse) {
+    var updateMacros = (bid, r) => {
+      return r ? r.replace(/\${AUCTION_PRICE}/g, parseFloat(bid.price)) : r;
```

Comment:
Since `replace` expects a string type fr the replacement, the parseFloat will be converted back to a string. Could you remove parseFloat or is it used to format the price?
`return r ? r.replace(/\${AUCTION_PRICE}/g, bid.price) : r;`